ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/Context.scala

```diff
@@ -0,0 +1,42 @@
+package com.daml.ledger.participant.state.kvutils.committer
+
+import com.daml.ledger.participant.state.kvutils.DamlKvutils.{
+  DamlLogEntry,
+  DamlStateKey,
+  DamlStateValue
+}
+import com.daml.ledger.participant.state.v1.ParticipantId
+import com.digitalasset.daml.lf.data.Time.Timestamp
+
+import scala.collection.mutable
+
+trait Context {
```

Comment:
Explain purpose of Context

Reply:
In particular explain what data it maintains and how it does so.

AFAIU, it maintains the inputs and a mutable state map on top of it; and the idea is to start with just the inputs and the other ledger metadata captured, and then mutate the outputs "into shape". Is that correct?

Reply:
Added explanation. Correct.